users/jarred/config.h

```diff
@@ -0,0 +1,55 @@
+/* Copyright 2018 Jarred Steenvoorden
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#ifndef USERSPACE_CONFIG_H
+#define USERSPACE_CONFIG_H
+
+#define PREVENT_STUCK_MODIFIERS
```

Comment:
This line is not actually needed, and should be removed.

Specifically, the build script automatically includes all of the config.h files, so this is not necessary.